buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -96,23 +90,25 @@ public String getScenarios() {
         return scenarios;
     }
 
+    @Internal
+    public Property<String> getForkPointCommitBaseline() {
+        return forkPointCommitBaseline;
+    }
+
     @Option(option = "baselines", description = "A comma or semicolon separated list of Gradle versions to be used as baselines for comparing.")
     public void setBaselines(@Nullable String baselines) {
-        this.baselinesProperty.set(baselines);
+        this.configuredBaselines.set(baselines);
     }
 
-    @Nullable @Optional @Input
+    @Optional @Input
     public String getBaselines() {
```

Comment:
Why do we still have this property? What is it used for?

Reply:
It was read by subclass `DistributedPerformanceTest`. Now it's removed.